hack/test-cmd.sh

```diff
@@ -327,13 +327,19 @@ for test in "${tests[@]}"; do
   echo
   echo "++ ${test}"
   name=$(basename ${test} .sh)
+  namespace="cmd-${name}"
 
+  os::test::junit::declare_suite_start "cmd/namespace-setup"
   # switch back to a standard identity. This prevents individual tests from changing contexts and messing up other tests
-  oc project ${CLUSTER_ADMIN_CONTEXT}
-  oc new-project "cmd-${name}"
+  VERBOSE=true os::cmd::expect_success "oc project ${CLUSTER_ADMIN_CONTEXT}"
+  VERBOSE=true os::cmd::expect_success "oc new-project '${namespace}'"
+  # wait for the project cache to catch up and correctly list us in the new project
+  VERBOSE=true os::cmd::try_until_text "oc project -q" "${namespace}"
```

Comment:
`oc project -q` never hits the server, it just echoes whatever namespace the current context is set to